src/services/database/message-fragment-repository.js

```diff
@@ -0,0 +1,24 @@
+import ModelFactory from '../../models';
+import { modelName } from '../../models/message-fragment';
+import { runWithinTransaction } from './helper';
+import { logInfo, logWarning } from "../../middleware/logging";
+
+const MessageFragment = ModelFactory.getByName(modelName);

```

Comment:
```suggestion
const MessageFragment = ModelFactory.getByName(modelName);

```
Should this variable not be camelCase ?

Reply:
It seems that PascalCase is the convention for models, would be good to get the others opinion though! @joefong-nhs @AndyFlintNHS